python/ray/serve/_private/logging_utils.py

```diff
@@ -74,6 +74,11 @@ def format(self, record: logging.LogRecord) -> str:
             ]
         if SERVE_LOG_ROUTE in record.__dict__:
             record_format[SERVE_LOG_ROUTE] = SERVE_LOG_RECORD_FORMAT[SERVE_LOG_ROUTE]
+
+        if "_extra" in record.__dict__ and record.__dict__["_extra"]:
+            for key in record.__dict__["_extra"]:
+                record_format[key] = f"%({key})s"
```

Comment:
Could you log a warning if the `key` already exists in `record_format`. That would help prevent users from overwriting existing keys.

Reply:
not applied, updated pr without using "_extra"